src/io/ctrees_utils.h

```diff
@@ -35,7 +36,7 @@ extern "C" {
     extern int assign_forest_ids(const int64_t ntrees, struct locations_with_forests *locations, int64_t *forests, int64_t *tree_roots);
     extern void sort_locations_on_fid_file_offset(const int64_t ntrees, struct locations_with_forests *locations);
     extern int fix_flybys(const int64_t totnhalos, struct halo_data *forest, struct additional_info *info, int verbose);
-    extern int fix_upid(const int64_t totnhalos, struct halo_data *forest, struct additional_info *info, int *interrupted, const int verbose);
+    extern int fix_upid(const int64_t totnhalos, struct halo_data *forest, struct additional_info *info, const int verbose);
```

Comment:
Signature will need updating after `ctrees_utils.c` is updated.

Reply:
Not fixing :D